src/components/NewClusterModal.js

```diff
@@ -216,209 +375,77 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
             ' to select another image.'
           ]),
           div({ style: { display: 'flex', justifyContent: 'flex-end' } }, [
-            h(ButtonSecondary, { style: { marginRight: '2rem' }, onClick: () => this.setState({ viewMode: undefined }) }, ['Back']),
+            h(ButtonSecondary,
+              {
+                style: { marginRight: '2rem' }, onClick: () => this.setState({ viewMode: undefined })
+              },
+              ['Back']),
             h(ButtonPrimary, { onClick: () => this.createCluster() }, ['Create'])
           ])
         ])
-      })
-    ], [
-      Utils.DEFAULT, () => ({
+      })],
+      [Utils.DEFAULT, () => ({
         onPrevious: undefined,
         contents: h(Fragment, [
-          div({ style: { marginBottom: '1rem' } }, [
-            'Choose a Terra pre-installed runtime environment (e.g. programming languages + packages) or choose a custom environment'
-          ]),
-          h(SimpleTabBar, {
-            tabs: [{ title: 'PRE-INSTALLED ENVIRONMENT', key: false, width: 265 }, { title: 'CUSTOM ENVIRONMENT', key: true, width: 215 }],
-            value: isCustomEnv,
-            onChange: value => this.setState({ isCustomEnv: value })
-          }),
-          div({
-            style: {
-              display: 'grid', gridTemplateColumns: '7rem 2fr 1fr', gridGap: '1rem', alignItems: 'center', margin: '1.5rem 0 1rem', minHeight: 100
-            }
-          }, [
-            isCustomEnv ?
-              h(Fragment, [
+          div({ style: { marginBottom: '1rem' } },
+            ['Create a compute instance to launch Jupyter Notebooks or a Project-Specific software application.']),
+          div({ style: { minHeight: 100 } },
```

Comment:
nit: This `minHeight` doesn't seem to be doing anything any more, so we can probably get rid of this div entirely.